x-pack/plugins/translations/translations/zh-CN.json

```diff
@@ -13991,165 +13989,6 @@
     "xpack.ml.fieldTypeIcon.numberTypeAriaLabel": "数字类型",
     "xpack.ml.fieldTypeIcon.textTypeAriaLabel": "文本类型",
     "xpack.ml.fieldTypeIcon.unknownTypeAriaLabel": "未知类型",
-    "xpack.ml.fileDatavisualizer.aboutPanel.analyzingDataTitle": "正在分析数据",
```

Comment:
I wonder if we can rename these to `xpack.fileDatavisualizer` instead of removing them completely 🤔

Reply:
that's just the way the string translation works, they have been renamed to `xpack.fileDataVisualizer` and will be replaced when the transitions are remade for release.